devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -133,7 +133,11 @@ def updated() {
 }
 
 def configure() {
-	// currently supported devices do not require initial configuration
+	//Recessed Door Sensor 7 - Enable Binary Sensor Report for S2 Authenticated
+	if (state.MSR == "0371-0102-00BB" || state.MSR == "0371-0002-00BB") {
```

Comment:
The more modern way to do this is to check the `zwaveInfo` object.

Could you update this to check `zwaveInfo.mfr == "0371" && zwaveInfo.model == "00BB"` instead?

Reply:
Changed the IF statement to use requested method.